src/Sulu/Bundle/WebsiteBundle/Routing/ContentRouteProvider.php

```diff
@@ -237,12 +237,14 @@ private function checkResourceLocator($resourceLocator, $resourceLocatorPrefix)
      */
     protected function getRedirectRoute(Request $request, $url)
     {
+        $format = 'html' !== $request->getRequestFormat() ? '.' . $request->getRequestFormat() : '';
```

Comment:
You never know which process a function is triggering or could trigger in future, so I would avoid a duplicated call of `$request->getRequestFormat()` and create a variable for it:

```
$requestFormat = $request->getRequestFormat();
$format = 'html' !== $requestFormat ? '.' . $requestFormat : '';
```

Reply:
Changed and rebased onto 1.6